### ISSUE TITLE
Why use pairwise pose prediction?

### ISSUE THREAD

**Issue by leo-frank** [QitaoZhao/E-RayZer#1]
Hi, happy to see this work, a step for 3D vision pretraining. 
I am curious about why not predict multiple camera poses at once like VGGT? Is there any consideration behind the choice?

**Comment by QitaoZhao** [comment]
Can check [this file](https://github.com/QitaoZhao/E-RayZer/blob/main/erayzer_core/utils/camera_eval.py)

**Comment by leo-frank** [comment]
Thanks for your reply again! 
I wonder the test split or json that you use to evaluate the camera pose and PSNR for DL3DV as shown in Table 1.
Is that same to [rayzer_evaluation_index_dl3dv_even.json](https://github.com/hwjiang1510/RayZer/blob/c34ca41e52e1725633228c56390d6ae5a3ba47b6/data/rayzer_evaluation_index_dl3dv_even.json) or [rayzer_evaluation_index_dl3dv_random.json](https://github.com/hwjiang1510/RayZer/blob/c34ca41e52e1725633228c56390d6ae5a3ba47b6/data/rayzer_evaluation_index_dl3dv_random.json) in Rayzer codes?

**Comment by leo-frank** [comment]
Got it. Really Thanks! 

I still have a question regarding the Multi-view Transformer used in the pose estimation stage.
In my opinion, introducing an attention mask into the camera-estimation transformer would better match the real-world scenario. Specifically, the camera prediction for the reference views should not be influenced by the target views, while the target views should still be able to attend to (and benefit from) the reference views.

I’d like to hear your thoughts on this suggestion.

**Comment by leo-frank** [comment]
Got it.  Thanks for your detailed reply. 
Another question is: how the DL3DV dataset is split into training and test set?
Looking forward to your reply.

**Comment by QitaoZhao** [comment]
Close for now due to inactivity; feel free to reopen!
